### DISCUSSION TITLE
Study hints at the existence of the closest black holes to Earth in the Hyades star cluster

### DISCUSSION THREAD

**Discussion by HanDeBruijn** [a-cosmology-group/acg#147] [Astrophysics and Physics]
Subtitle: _**Black holes** are one of the most mysterious and fascinating phenomena in the Universe_.

"Our _simulations_ can only simultaneously match the mass and size of the Hyades if some _black holes_ are present at the centre of the cluster today (or until recently)", says the first author of **[the paper](https://www.eurekalert.org/news-releases/1000958)**.
Likewise, simulations can only match the rotation curve of a galaxy if a halo of _dark matter_ is present around it.

Is there a pattern?

**Comment by bgreenman** [comment]
The halo might consist ofcluster of large mass dust and gas as an alternative

**Comment by RedshiftDrift** [comment]
Same pattern as for the Big Bang model; people incorrectly believe that if a simulations match observations, the hypotheses used in the simulations must be true.

**Reply by bgreenman** [reply]
The main problem is there should be a connection between simulation and experiment. first comes the simulation and then the simulation is tested in an experiment, that is the right procedure,but many researchers don't test there simulations
